src/neutron/libGudrunN/groups_routines.f90

```diff
@@ -463,6 +464,14 @@ subroutine get_groups(ignore,wavemin,wavemax,outputunitstype)
         close (10)
 !c Write a file to store group calibration suitable to be read by Gudrun_GUI
             fname1=fname1(1:len_trim(fname1))//'.cal'
+            start=INDEX(fname1, "/", BACK=.TRUE.)
+            IF (start == 0 ) THEN
+                start=INDEX(fname1, "\", BACK=.TRUE.)
+            END IF
+            IF (start > 0) THEN
+                start=start+1
+            END IF
+            fname1=fname1(start:len_trim(fname1))
```

Comment:
In the interests of preservation (and safety!) I think any modifications we make should be conditionally-compiled and *not* be the default. So:
```suggestion
#ifdef GUDPY_COMPATIBILITY
            start=INDEX(fname1, "/", BACK=.TRUE.)
            IF (start == 0 ) THEN
                start=INDEX(fname1, "\", BACK=.TRUE.)
            END IF
            IF (start > 0) THEN
                start=start+1
            END IF
            fname1=fname1(start:len_trim(fname1))
#endif
```
We would then enable this additional code by passing `-DGUDPY_COMPATIBILITY=1` to `cmake` at configure time. I believe that we also need to add `-DCMAKE_Fortran_FLAGS:string="-cpp"` to enable the C preprocessor.